### DISCUSSION TITLE
software/2023/11/06/cms

### DISCUSSION THREAD

**Discussion by giscus** [SuperSharkler/supersharkler.github.io#17] [Github-Page-Comments]
# software/2023/11/06/cms

In the past week, I have been exploring different options to add the commenting feature to this site. And the conclusion is, we need to make our site dynamic, meaning we need a backend to store and display the comments. This blog will discuss two possible solutions I found.

https://robosharkler.github.io/software/2023/11/06/cms.html

**Comment by robosharkler** [comment]
this is a test comment 😊
